### PR TITLE
[TRAFODION-2866] Add HBase Configuration Changes for *Trafodion Provisioning Guide*

### PR DESCRIPTION
**Pull request by liuyu000**

### CODE REVIEW

File: docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc [hunk 1]
```diff
@@ -331,13 +331,38 @@ default. Depending on the size of a user table, we have experienced timeout fail
...
+When you have enough memory, you can increase this value to 7 so that more data can be temporarily accepted before flushing to disk instead of blocking writes.
+|This property blocks any further writes from clients to memstores if the memstores exceed the value of `multiplier * flush size`.
+
+Default value: 2
+| hbase.hregion.memstore.flush.size | 536870912 bytes | HBase uses memstore to buffer data before writing it to disk. Once the data in memstore has outgrown this size, it is flushed as an HFile to disk.
+
+Default value: 134217728 bytes (128M)
+| hbase.hstore.blockingStoreFiles | 200 | http://gbif.blogspot.com/2012/07/optimizing-writes-in-hbase.html
+
+This property blocks any further writes from memstores to regions after HFile number hitting this limit until compactions are completed.
```
Comment (traflm) [inline docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc]: This property blocks any further writes from memstores to regions after HFile number hitting this limit until compactions are completed.
More precisely, may be:
This property blocks any further writes from memstores to HFile, after existing HFile number hitting this limit until compactions are completed.

Comment (liuyu000) [reply docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc]: Thanks Ming, your comment has been incorporated. :)

File: docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc [hunk 2]
```diff
@@ -331,13 +331,38 @@ default. Depending on the size of a user table, we have experienced timeout fail
...
+| hbase.snapshot.master.timeoutMillis 
+
+and 
+
+hbase.snapshot.region.timeout | 10 minutes | HBase's default setting is 60000 milliseconds. 
 If you experience timeout issues with HBase snapshots when you use the {project-name} Bulk Loader or other statements, 
 you can set the value for these two HBase properties to 10 minutes (600,000 milliseconds).
-| hbase.hregion.max.filesize | 107374182400 bytes | HBase's default setting is 10737418240 (10 GB). We have increased the setting to 
-107374182400 (100 GB), which reduces the number of HStoreFiles per table and appears to reduce disruptions to active transactions from 
+| hbase.hregion.max.filesize | 107374182400 bytes | HBase's default setting is 10737418240 bytes (10 GB). You can increased the setting to
```
Comment (DaveBirdsall) [inline docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc]: Grammar typo: "You can increased" should be "You can increase"

File: docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc [hunk 3]
```diff
@@ -331,13 +331,38 @@ default. Depending on the size of a user table, we have experienced timeout fail
...
+When you have enough memory, you can increase this value to 7 so that more data can be temporarily accepted before flushing to disk instead of blocking writes.
+|This property blocks any further writes from clients to memstores if the memstores exceed the value of `multiplier * flush size`.
+
+Default value: 2
+| hbase.hregion.memstore.flush.size | 536870912 bytes | HBase uses memstore to buffer data before writing it to disk. Once the data in memstore has outgrown this size, it is flushed as an HFile to disk.
+
+Default value: 134217728 bytes (128M)
+| hbase.hstore.blockingStoreFiles | 200 | http://gbif.blogspot.com/2012/07/optimizing-writes-in-hbase.html
+
+This property blocks any further writes from memstores to HFile, after existing HFile number hitting this limit until compactions are completed.
```
Comment (DaveBirdsall) [inline docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc]: Suggested wordsmith: "This property blocks any further writes from memstore to HFiles, after the number of existing HFiles hits this limit, until compactions are completed."

Comment (liuyu000) [reply docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc]: Thanks Dave, your comment has been incorporated :)

File: docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc (hunk 2 above)
Comment (liuyu000) [reply docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc]: Oops, sorry, I've corrected, thanks Dave :)

**Pull request merged**